### DISCUSSION TITLE
Clarification on Crazyflie 2.0 Prototyping Deck: Pin Connections for Servo Motor Attachment

### DISCUSSION THREAD

**Discussion by NarekHarutyu** [bitcraze/discussions#774] [Q&A]
Hello, I have some questions related to attaching a servo motor to the Crazyflie 2.0 board using the Prototyping deck. Initially, I attempted to connect the servo's wires directly to the pins of the Crazyflie. I connected VCOM, GND, and one of the IOs, which caused the servo to start spinning. However, when I placed the Prototyping deck on the board and tried to attach the servo, it stopped working.

I would like to know which holes on the Prototyping deck are connected to the pins. Specifically, I'm interested in understanding how they are connected in the schematic. Are the holes already connected, or do I need to add additional wires to establish connections between the hole where the pins come through and the ones I intend to use for connecting the servo?

I would greatly appreciate it if you could provide me with a better understanding of how the Prototyping deck functions and how the holes in the middle are connected to the pin holes. I couldn't find this information in the schematics available on your website.

**Comment by ataffanel** [comment]
Hi,

The prototyping deck is designed to be a base-board for hardware project. It is very similar to a [PerfBoard](https://en.wikipedia.org/wiki/Perfboard) which is a common electronic development tool.

You can find the cabling of the board in the [specification page](https://www.bitcraze.io/products/prototyping-deck/):
![image](https://github.com/bitcraze/discussions/assets/6381499/e1cfa961-2fbd-454e-a7a6-7537287310e5)

The basic idea is that you can solder connector and other electronic on the plated holes of the deck, the row A and F of holes are themselves connected to pins of the expansion port, so you can use solder-bridge or small wires to connect the power coming from the expansion port to your connector. One example of usage of this prototyping deck is the old [flying traffic light](https://www.bitcraze.io/wp-content/gallery/crazyflie-2-0-expansion-port/cf2_trafficlight_3.jpg) we did to demonstrate the concept.
